alternator/serialization.cc

```diff
@@ -161,32 +161,37 @@ bytes get_key_column_value(const rjson::value& item, const column_definition& co
     return get_key_from_typed_value(*key_typed_value, column);
 }
 
-// Parses the JSON encoding for a key value, which is a map with a single
-// entry, whose key is the type (expected to match the key column's type)
-// and the value is the encoded value.
-bytes get_key_from_typed_value(const rjson::value& key_typed_value, const column_definition& column) {
+static const rjson::value& get_typed_value(const rjson::value& key_typed_value, std::string type_str, std::string_view name) {
```

Comment:
nitpick: I know that the previous code is based on std::string (historical reasons, probably), but perhaps we could just use string_view here? In this function you don't really need to claim ownership of type_str for any reason.

Reply:
I think I used `std::string_view` initially but had problems with `rjson`. I will try this again.